indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,23 @@
         }
     }
 }
+
+#assign-role-table {
```

Comment:
The problem is maintaining such components. If we're speaking of things that are clearly reusable (e.g. a table), yeah, that seems fair enough, however.